src/NUnitFramework/framework/Constraints/EqualStringWithoutUsingConstraint.cs

```diff
@@ -20,6 +20,7 @@ public class EqualStringWithoutUsingConstraint : Constraint
 
         private bool _caseInsensitive;
         private bool _ignoringWhiteSpace;
+        private bool _NormalizeLineEndings;
```

Comment:
I guess I'll have to fix this copypasta. I can do this after the first round of reviews.
```suggestion
        private bool _normalizeLineEndings;
```

Reply:
Should have shown IDE1006 warning in your editor:
![{7F4443CF-5250-4290-BD7F-C848A4AECFFD}](https://github.com/user-attachments/assets/25917002-66ee-4c56-97e5-4e7049e87206)

Reply:
I did a non-case-preserving string replacement globally right before pushing. It does flag that, but it doesn't block a build because [`EnforceCodeStyleInBuild`](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#enforcecodestyleinbuild) is not set.

Reply:
Huh, interestingly https://github.com/nunit/nunit/blob/d70b7eab12c41da68ccc313fd5d3d5abcdf15fa4/src/NUnitFramework/Directory.Build.props#L9 _is_ set.